File: FutsalSystem/FutsalSystem/Models/DTO/Match/MatchDTO.cs
using System;
using System.Collections.Generic;

namespace FutsalSystem.Models.DTO.Match
{
    public class MatchDTO
    {
        public int Id { get; set; }
        public int HomeTeamId { get; set; }
        public string HomeTeam { get; set; }
        public int AwayTeamId { get; set; }
        public string AwayTeam { get; set; }
        public DateTime MatchDate { get; set; }
        //public int Round { get; set; } // Turas
        public int HomeTeamFirstHalfScore { get; set; }
        public int AwayTeamFirstHalfScore { get; set; }
        public int HomeTeamScore { get; set; }
        public int AwayTeamScore { get; set; }
        public bool IsFinished { get; set; }
        public IEnumerable<Models.MatchEvent> MatchEvents { get; set; }
    }
}

File: FutsalSystemFront/src/app/Models/MatchDto.ts
import { MatchEventDto } from './MatchEventDto';

export class MatchDto {
    public Id: number;
    public HomeTeamId: number;
    public HomeTeam: string;
    //public HomeTeam: TeamDto;

    public AwayTeamId: number;
    public AwayTeam: string;
    //public AwayTeam: TeamDto;

    public MatchDate: Date;
    public HomeTeamFirstHalfScore: number;
    public AwayTeamFirstHalfScore: number;
    public HomeTeamScore: number;
    public AwayTeamScore: number;
    public IsFinished: boolean;
    public MatchEvents: MatchEventDto[];

    constructor(id: number, homeTeamId: number, awayTeamId: number, homeTeam: string, awayTeam: string, matchDate: Date, homeTeamFirstHalfScore: number, awayTeamFirstHalfScore: number, homeTeamScore: number, awayTeamScore: number, isFinished: boolean, matchEvents: MatchEventDto[]) {
        this.Id = id;
        this.HomeTeamId = homeTeamId;
        this.AwayTeamId = awayTeamId;
        this.HomeTeam = homeTeam;
        this.AwayTeam = awayTeam;
        this.MatchDate = matchDate;
        this.HomeTeamFirstHalfScore = homeTeamFirstHalfScore;
        this.AwayTeamFirstHalfScore = awayTeamFirstHalfScore;
        this.HomeTeamScore = homeTeamScore;
        this.AwayTeamScore = awayTeamScore;
        this.IsFinished = isFinished;
        this.MatchEvents = matchEvents;
    }
}